.mocharc.cjs

```diff
@@ -7,6 +7,6 @@ module.exports = {
   extension: ['js', 'spec.cjs', 'spec.mjs', 'spec.ts', 'spec.mts'],
   require: ['mocha-expect-snapshot/old-format', 'test/mocha.config.cjs'],
   // Resolve absolute path for test with fork and different cwd.
-  loader: require.resolve('@esbuild-kit/esm-loader'),
+  loader: require.resolve('tsx'),
```

Comment:
I don't see using tsx as a loader as an issue but I understand your reasoning and removed it. Using the loaders by itself looks quite convoluted.

What command should be run in the cli tests now? This obviously doesn't work without tsx: https://github.com/jhipster/generator-jhipster/blob/4ce31cec6fb5fc010f69206b549fffa25d813dea/test/utils/utils.cjs#L71
I already tried ``` let cmd = `node --require @esbuild/cjs-loader --loader @esbuild-kit/esm-loader ${cmdPath} `;``` but it does not work.

I think using tsx as a replacement for node in the tests similar to what you would do with ts-node (and like it was done in the cli test) could also be a good solution. With that we could also get rid of the warning: `(node:97101) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time` which is quite annoying. I could not get that to work though. And this would mean we need to keep tsx.

Reply:
I think we can use the cli from `dist` folder instead of using tsx.
It's just basic tests like `--version`, `unknown command` and a few more.
What do you think?